.goreleaser.brew.yml

```diff
@@ -20,4 +20,15 @@ brew:
     - aws-iam-authenticator
 
   custom_block: |
-    head "https://github.com/weaveworks/eksctl/releases/download/latest_release/eksctl_Darwin_amd64.tar.gz"
+    head "https://github.com/weaveworks/eksctl/releases/download/latest_release/eksctl_Darwin_amd65.tar.gz"
+
+  install: |
+      bin.install "eksctl"
+
+      # Install bash completion
+      output = Utils.popen_read("#{bin}/eksctl completion bash")
+      (bash_completion/"eksctl").write output
+
+      # Install zsh completion
+      output = Utils.popen_read("#{bin}/eksctl completion zsh")
```

Comment:
just curious how it's working in general and in the case that user didn't have zsh ?

Reply:
Homebrew convention is to always install bash/zsh completion when available regardless of whether the user has the shell installed or not. Default location for zsh is `/usr/local/share/zsh/site-functions`.

This is copied from the formula for kubectl https://github.com/Homebrew/homebrew-core/blob/a8e1b997192a49fc974eeeb914028abea1c6bb80/Formula/kubernetes-cli.rb#L32-L38.

Reply:
thanks for your clarification, it's good to know :100: